scripts/get-shapefiles.py

```diff
@@ -0,0 +1,387 @@
+#!/usr/bin/env python
+
+# This script generates and populates the 'data' directory with all needed
+# shapefiles.
+
+from __future__ import (
+    division,
+    absolute_import,
+    print_function,
+    unicode_literals)
+import os
+import errno
+import tarfile
+import zipfile
+import subprocess
+import distutils.spawn
+import argparse
+import sys
+import tempfile
+import logging
+import time
+import email.utils
+
+if sys.version_info >= (3,):
+    import urllib.request as urllib2
+    import urllib.parse as urlparse
+else:
+    import urllib2
+    import urlparse
+
+data_dir = 'data'
+settings = {
+    # Keys 1, 2, 3, ... set the arg short-options and the related process
+    # ordering. Use > 0 to allow processing.
+    1: {
+        'directory': 'world_boundaries',
+        'url': 'http://planet.openstreetmap.org/historical-shapefiles/world_boundaries-spherical.tgz',  # noqa
```

Comment:
#noqa special comment disables checks made by the pep8 script at line level: I only used it to confirm long URLs.